src/js/views/patients/worklist/worklist_views.js

```diff
@@ -186,8 +187,25 @@ const GroupsDropList = Droplist.extend({
   },
 });
 
+const SortDropList = Droplist.extend({
+  popWidth: 248,
+  picklistOptions() {
+    return {
+      headingText: intl.patients.worklist.worklistViews.sortDropList.headingText,
+      itemTemplate: hbs`
```

Comment:
I think this is essentially the default `itemTemplate`.  Is it needed?

Reply:
nope, sure isn't